src/main/java/com/github/manolo8/darkbot/gui/drawables/StatsDrawer.java

```diff
@@ -0,0 +1,152 @@
+package com.github.manolo8.darkbot.gui.drawables;
+
+import com.github.manolo8.darkbot.core.objects.facades.BoosterProxy;
+import com.github.manolo8.darkbot.core.objects.group.GroupMember;
+import eu.darkbot.api.config.ConfigSetting;
+import eu.darkbot.api.config.types.DisplayFlag;
+import eu.darkbot.api.extensions.Drawable;
+import eu.darkbot.api.extensions.Feature;
+import eu.darkbot.api.extensions.MapGraphics;
+import eu.darkbot.api.game.other.Point;
+import eu.darkbot.api.managers.BoosterAPI;
+import eu.darkbot.api.managers.ConfigAPI;
+import eu.darkbot.api.managers.GroupAPI;
+import eu.darkbot.api.managers.RepairAPI;
+import eu.darkbot.api.managers.StatsAPI;
+import eu.darkbot.util.TimeUtils;
+
+import java.awt.font.TextAttribute;
+import java.text.DecimalFormat;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.ToIntFunction;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Feature(name = "Stats Drawer", description = "Draws statistics")
+public class StatsDrawer implements Drawable {
```

Comment:
```suggestion
@Feature(name = "Overlay Drawer", description = "Draws overlays (eg: stats, boosters, or group info)")
public class OverlayDrawer implements Drawable {
```
I'd rename this class to OverlayDrawer, i think it better represents what this is